File: C/CLMPMTPST/CLMPMTPST/CLMPMTPST/Data/Payment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;

namespace CLMPMTPST
{
	public class Payment
	{
		public const string DATE_FORMAT = "MMddyy";
		public int ClaimPaymentId { get; set; }
		public string AccountNumber { get; set; }
		public string Ssn { get; set; }
		public double PaymentAmount { get; set; }
		[DisplayName("Effective Date")]
		public DateTime EffectiveDate { get; set; }
		[DisplayName("Guarantor Code")]
		public string GuarantorCode { get; set; }
		[DisplayName("Loan Sequences")]
		public List<int> LoanSequences { get; set; }
		[DisplayName("Last Name")]
		public string LastName { get; set; }

		public Payment(string ssn, double amount, DateTime effectiveDate, string guarantorCode, List<int> loanSequences, string lastName)
		{
			Ssn = ssn;
			PaymentAmount = amount;
			EffectiveDate = effectiveDate;
			GuarantorCode = guarantorCode;
			LoanSequences = loanSequences;
			LastName = lastName;
		}
    }
}

File: M/MD/MDBSHome/Payment.vb
Public Class Payment
    Private _amount As Double = 0
    Public Property Amount() As Double
        Get
            Return _amount
        End Get
        Set(ByVal value As Double)
            _amount = value
        End Set
    End Property

    Private _receivedDate As String = ""
    Public Property ReceivedDate() As String
        Get
            Return _receivedDate
        End Get
        Set(ByVal value As String)
            _receivedDate = value
        End Set
    End Property
End Class
